lib/jsmidgen.js

```diff
@@ -485,6 +485,31 @@ var Midi = {};
 	};
 
 	/**
+	 * Add a note-on and -off event to the track for each pitch in an array of pitches.
+	 *
+	 * @param {number} channel - The channel to add the event to.
+	 * @param {array} chord - An array of pitches, either numeric or
+	 * symbolic.
+	 * @param {number} dur - The duration of the chord, in ticks.
+	 * @param {number} [velocity=90] - The velocity of the chord,
+	 * defaults to DEFAULT_VOLUME.
+	 * @returns {Track} The current track.
+	 */
+	Track.prototype.addChord = Track.prototype.chord = function(channel, chord, dur, velocity) {
+		if (!Array.isArray(chord)) {
+			throw new Error('Chord must be an array of pitches');
+		}
+		for (var i = 0; i < chord.length; i++) {
+			this.noteOn(channel, chord[i], velocity);
```

Comment:
I m off the understanding that this function will need to have the third argument only for the first note. If you add this for each note, then it may result in an unexpected outcome in the MIDI file itself. It d be good to add a test or confirm with @dingram about this.